docs/source/bag-creation.rst

```diff
@@ -65,3 +65,53 @@ attached directly to bags with ``.str.methodname``:
 .. code-block:: python
 
    >>> b = db.from_filenames('myfile.*.csv.gz').str.strip().str.split(',')
+
+
+``db.from_imperative``
+----------------------
+
+You can construct a dask bag from :doc:`dask.delayed<imperative.rst>` values
```

Comment:
Broken reference, `dask.delayed<imperative.rst>` should be `dask-delayed <imperative>` (with space).